frontend/src/routes/Home/Home.js

```diff
@@ -1,17 +1,47 @@
 import AppLogo from 'components/AppLogo'
+import { Link } from 'react-router-dom'
 import './Home.scss'
+import PATH from '../../utils/path'
 
 function Home() {
   return (
     <div className="Home">
       <header className="Home__header">
         <AppLogo />
+
         <p>
           Edit <code>codebase</code> and save to reload.
         </p>
+
         <a className="Home__link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
           Learn React
         </a>
+
+        <p>Temp router links</p>
+
+        <ul>
+          <li>
+            <Link to={PATH.DATASET.replace(':', '1')}>DATASET</Link>
```

Comment:
Storybook fails here, and will for any other page we add a Link to in the future, with the error `useHref() may be used only in the context of a <Router> component.`. That can be resolved by wrapping `<Home {...args} />` in a MemoryRouter.

Reply:
Fixed using a decorator